lib/swoosh/adapters/mailgun.ex

```diff
@@ -248,7 +248,8 @@ defmodule Swoosh.Adapters.Mailgun do
 
   defp prepare_template_options(body, _), do: body
 
-  defp encode_body(%{attachments: attachments} = params) do
+  @doc false
```

Comment:
I had to make it public because `bypass` will receive the request built by Hackney so the new assertions will always pass, so we need an extra test to make sure the value is at least built properly.